src/ChatRouter.php

```diff
@@ -18,17 +19,17 @@ public function chat(): Closure
             ChatRouter::$isBlockedBaseRoutes = true;
 
             $defaultOptions = [
-                'conversations_search' => false,
-                'conversations_delete' => false,
-                'conversations_get' => false,
-                'conversations_get_by_user' => false,
-                'messages_search' => false,
-                'messages_create' => false,
-                'messages_read' => false,
+                'conversations_search' => true,
+                'conversations_delete' => true,
+                'conversations_get' => true,
+                'conversations_get_by_user' => true,
+                'messages_search' => true,
+                'messages_create' => true,
+                'messages_read' => true,
             ];
 
-            foreach ($options as $option) {
-                $defaultOptions[$option->value] = true;
+            if (!empty($options)) {
+                $defaultOptions = Arr::map($defaultOptions, fn ($value, $defaultOption) => !empty(Arr::first($options, fn ($option) => $option->value === $defaultOption)));
```

Comment:
let's wrap $options to collection, it allows to use the `contains` helper

```suggestion
                $defaultOptions = Arr::map($defaultOptions, fn ($value, $defaultOption) => $options->contains('value', $defaultOption));
```